### PR TITLE
Fixes modify variables interpreting dirs wrongly

### PR DESCRIPTION
**Pull request by Cyberboss**

### CODE REVIEW

File: code/modules/admin/verbs/massmodvar.dm [hunk 1]
```diff
@@ -64,7 +64,8 @@
 
 	if(default == VV_NUM)
 		var/dir_text = ""
-		if(dir < 0 && dir < 16)
+		var/dir = var_value
```
Comment (RemieRichards) [inline code/modules/admin/verbs/massmodvar.dm]: come on you know what's wrong here.

**Review by RemieRichards — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/massmodvar.dm (hunk 1 above)
Comment (RemieRichards) [reply code/modules/admin/verbs/massmodvar.dm]: locals shouldn't take the name of object vars

Comment (Cyberboss) [reply code/modules/admin/verbs/massmodvar.dm]: Oh, ya, I knew that, but I'm lazy af

Comment (RemieRichards) [reply code/modules/admin/verbs/massmodvar.dm]: *growl*

**Pull request merged**